aten/src/ATen/core/SymInt.h

```diff
@@ -48,6 +49,8 @@ class TORCH_API SymInt {
             return data_ + sci.data_;
         }
 
+        c10::SymbolicIntNode* toSymbolicIntNode();
+
         int64_t data() const {
```

Comment:
We shouldn't expose data directly, since it isn't really an `int64_t`, it might also be some masked nonsense. Also, standard practice in C++ is when we're doing bit-masky stuff, we should represent it as a `uint64_t`, not a signed integer; signed integers are undefined with bitwise operators, see https://wiki.sei.cmu.edu/confluence/display/c/INT13-C.+Use+bitwise+operators+only+on+unsigned+operands

Reply:
Sure, I can switch to uint64_t. I was trying to avoid conversions from int64_t to uint64_t since that's the type we are using for dimension values and we aren't doing any "dangerous" bit-wise stuff as shifting.